www/js/toolbar.js

```diff
@@ -128,9 +140,7 @@ const setListeners = () => {
             isPinCorrect ? 'Connection Succesful' : 'Connection Failed',
           );
         },
-        /* Coming soon
-        'bani' : global.core.search.loadBani(data.baniId, data.verseId);
-        */
+        bani: payload => loadBani(payload.baniId, payload.verseId),
```

Comment:
I need to spend some draw this architecture out. I'm losing track of how everything is loaded 😓